### ISSUE TITLE
GIFs doesn't need preview (or need a different kind of preview)

### ISSUE THREAD

**Issue by celiamgtt** [algoo/preview-generator#201]
It's not really interesting to have a GIF preview. The interesting thing about GIFs is that they're animated. I would expect GIFs stay animated.

**Comment by lebouquetin** [comment]
you right @celiamgtt . Do you know if there is an issue on https://github.com/tracim/tracim?

There are two ways to fix this:

- change the way preview-generator is used in tracim
- implement a "best" preview mechanism in preview-generator (currently, the strategy is statically defined as "get_jpeg_preview")

**Comment by inkhey** [comment]
I'm not sure we do not need a preview for gif.
If the size (width/length) of the original file is too high, we can resize using css (is there a limit ?) but what should happened if the gif is a very big file ? Should we force the browser to load a big gif ? Is there any magic in the browser to load it gracefully like for video ?

Another solution may be to provide gif preview by resizing gif, there is already an issue about gif preview for video: "https://github.com/algoo/preview-generator/issues/177

**Comment by inkhey** [comment]
It seems like the way to go to save bandwidth is converting gif to video : https://www.smashingmagazine.com/2018/11/gif-to-video/

**Comment by celiamgtt** [comment]
@lebouquetin Yes I created both issues as Philippe told me to do so but I forgot to link the 2 issues x)
